de.j0chn.nextcloud_password_client.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Name=Nextcloud Password Client
+Comment=Nextcloud Password Manager
+Icon=de.j0chn.nextcloud_password_client.nextcloud_password_client
+Exec=env GDK_GL=gles nextcloud_password_client %U
```

Comment:
exec is supposed to be the exec name + parameter. Also `gles` for x86_64 only? That's kinda ironic.

Reply:
I will create a arm64 build near future. So this is for later releases

Reply:
no this is for the current ony, there is only one .desktop file.

Reply:
Aleight. Then I will delete this and add it later then.

Reply:
there is one in the upstream repository.

.desktop aren't specific to snap nor flatpak, they are also used for distro packages.